backend/tests/schemas/test_generation.py

```
@@ -0,0 +1,313 @@
+        )
+
+        assert config.resolution is None
+        assert config.fps is None
+        assert config.max_duration is None
+
+
+class TestModelSpecWithImageConfig:
+    """Tests for ModelSpec with imageConfig field."""
+
+
+class TestModelSpecWithImageConfig:
```

Ruff (0.15.2)

[error] Redefinition of unused `TestModelSpecWithImageConfig` from line 251: `TestModelSpecWithImageConfig` redefined here

(F811)